llpc/context/llpcCompiler.cpp

```diff
@@ -1259,9 +1265,32 @@ Result Compiler::buildPipelineInternal(Context *context, ArrayRef<const Pipeline
 
   context->setDiagnosticHandlerCallBack(nullptr);
 
+  if (result == Result::Success) {
+    const int llpcVersion[2] = {LLPC_INTERFACE_MAJOR_VERSION, LLPC_INTERFACE_MINOR_VERSION};
+    auto hash = context->getPipelineContext()->getPipelineHashCodeWithoutCompact();
+    addHashSectionToElf(*pipelineElf, StringRef(reinterpret_cast<const char *>(hash.bytes), sizeof(hash)),
+                        StringRef(reinterpret_cast<const char *>(llpcVersion), sizeof(llpcVersion)));
```

Comment:
I'd pass these two as `ArrayRef`s, because you treat them as data and not as strings, but it doesn't matter much.

Reply:
done